### PR TITLE
Add filter list to VisitGap basis function for image survey; Fix bug …

### PR DESCRIPTION
**Pull request by edennihy**
…with hard-coded visit_gap value for image survey

Black

### CODE REVIEW

File: python/lsst/ts/fbs/utils/auxtel/basis_functions.py [hunk 1]
```diff
@@ -61,6 +63,10 @@ def get_basis_functions_image_survey(
     note_interest : `str`
         A substring that maps to surveys to be accounted for against the
         reference number of observations.
+    filter_names : list [str]
```
Comment (tribeiro) [inline python/lsst/ts/fbs/utils/auxtel/basis_functions.py:66]:
```
list [str] -> `list` [ `str` ]
```

**Review by tribeiro — APPROVED** [review]
Please, update the version history before merging!

File: conda/conda_build_config.yaml [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-  - 1.20
 rubin_sim:
-  - 0.10
-  - 0.8
+  - 0
 zip_keys:
   - python
   - numpy
   - rubin_sim
+
```
Comment (tribeiro) [inline conda/conda_build_config.yaml:11]: Add new empty line please!

**Comment by edennihy** [comment]
Requested changes incorporated, version history updated to v0.2.1

File: conda/conda_build_config.yaml (hunk 2 above)
Comment (edennihy) [reply conda/conda_build_config.yaml]: Ooops, grabbed this as part of [7485a63]

**Pull request merged**